draft-davies-internal-tld.mkd

```diff
@@ -33,51 +37,80 @@ informative:
     title: "Identification of a top-level domain for private use"
     target: https://itp.cdn.icann.org/en/files/root-system/identification-tld-private-use-24-01-2024-en.pdf
     date: 2024-01
+  ICANN-Board-Resolution:
+    title: "Reserving .INTERNAL for Private-Use Applications"
+    target: https://www.icann.org/en/board-activities-and-meetings/materials/approved-resolutions-special-meeting-of-the-icann-board-29-07-2024-en#section2.a
+    date: 2024-07
 
 --- abstract
 
-This document describes the reservation of the "internal" top-level domain for
+This document describes the reservation of the ".internal" top-level domain for
```

Comment:
Note this is not consistent with other RFCs, e.g. most recently RFC 9121, we do not use the dot. While it is arguably a matter of test, elsewhere where it refers to string/label the dot is certain not part of those.

Reply:
There's no consistency.

RFC6761, Section 6.2 -> test. / .test / .test.
RFC6762 -> .local.
RFC8375 -> home.arpa.
RFC9121 -> int
RFC9476 -> .alt